### ISSUE TITLE
False positive on B902

### ISSUE THREAD

**Issue by hmvp** [PyCQA/flake8-bugbear#74]
[Pydantic validator](https://pydantic-docs.helpmanual.io/#validators) decorators make classmethods from the decorated methods but these are flagged as instance methods

The following gets flagged:
```
class UserModel(BaseModel):
    name: str

    @validator('name')
    def name_must_contain_space(cls, v):
        if ' ' not in v:
            raise ValueError('must contain a space')
        return v.title()
```

changing `cls` to `self` gives no error

**Comment by boarik** [comment]
I believe this is B902. I'm experiencing the same issue.
(related issue in a pylint+pydantic context https://github.com/samuelcolvin/pydantic/issues/568)

**Comment by DylanYoung** [comment]
Here's how pep8-naming handles it: https://github.com/PyCQA/pep8-naming#options
